app/controllers/student_learning_resources_controller.rb

```diff
@@ -17,8 +18,18 @@ def create
     end
   end
 
+  def destroy
+    student_id = @student_learning_resource[:user_id]
```

Comment:
I don't see the need of treating `@student_learning_resource` as a hash 
If this is an object then you better send it a message 
```
student_id = @student_learning_resource.student_id
```

Reply:
Actually, that line is not required